src/main/resources/templates/post/show.ftl

```diff
@@ -1,5 +1,6 @@
 <#import '../include/categories.ftl' as c >
 <#include "../include/header.ftl" >
+<#include "../include/facebookSocialPlugin.ftl">
```

Comment:
Maybe, no need to extract it to partial template?

Reply:
I think it's better for future development, if we need to add Facebook plugin on other pages